config.go

```diff
@@ -642,6 +650,7 @@ type ConfigTemplate struct {
 	Backup         bool          `json:"backup" mapstructure:"backup"`
 	LeftDelim      string        `json:"left_delimiter" mapstructure:"left_delimiter"`
 	RightDelim     string        `json:"right_delimiter" mapstructure:"right_delimiter"`
+	Wait           *watch.Wait   `json:"wait" mapstructure:"wait"`
```

Comment:
Yep - those are covered down in here - https://github.com/hashicorp/consul-template/blob/master/watch/wait_test.go.